src/linter/html/transpiler.ts

```diff
@@ -161,6 +202,45 @@ function lintBootstrapAttributes(tag: Tag, report: HtmlReporter) {
 	}
 }
 
+function checkPreloadAttr(attr: Attribute, report: HtmlReporter) {
+	const value = attr.value.value.toLowerCase();
+	if (value && !["auto", "async", "sync"].includes(value)) {
+		report.addMessage(MESSAGE.REDUNDANT_BOOTSTRAP_PARAM_ERROR, {
+			name: attr.name.value,
+			messageDetails: "Use sap-ui-debug=true to suppress library preload requests",
+		}, attr.name);
+	}
+}
+
+function checkOriginInfoAttr(attr: Attribute, report: HtmlReporter) {
+	if (attr.value.value.toLowerCase() === "true") {
+		report.addMessage(MESSAGE.REDUNDANT_BOOTSTRAP_PARAM_ERROR, {
```

Comment:
The origin info is deprecated and will be removed, so I would have expected it to be abandoned if used.

Reply:
Thanks!